CRM/EventsExtras/Hook/BuildForm/EventFee.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Class for Event Fee BuildForm Hook
+ */
+class CRM_EventsExtras_Hook_BuildForm_EventFee extends CRM_EventsExtras_Hook_BuildForm  {
+  
+  /**
+  *
+  * @param string $$eventTab
+  */
+  public function __construct($eventTab) {
+    parent::__construct($eventTab);
+  }
+
+  /**
+  *
+  * @param \CRM_Event_Form_ManageEvent_Fee $form
+  */
+  public function buildForm(CRM_Event_Form_ManageEvent_Fee &$form) {
+    $this->hideField($form);
+  }
+
+}
```

Comment:
Can you configure your IDE to add a newline at end of file? Maybe you need to add a .editorconfig file to your project. I have a sample one If you need.

Reply:
@tunbola  please share with me thanks.